src/sigstore.ts

```diff
@@ -43,7 +43,9 @@ export type SignOptions = {
   oidcClientSecret?: string;
 } & TLogOptions;
 
-export type VerifierOptions = TLogOptions;
+export type VerifierOptions = {
+  findKey?: FindKeyFunc;
```

Comment:
this is a very small nitpick, `find` sounds somehow out of place.. not entirely sure why, but wondering if `getKey` sounds better, or `getPublicKey` even? Like from the libs perspective, it's getting the key not finding it?

Reply:
Good call. Updated!